src/components/_inputs/TextField/TextField.stories.tsx

```diff
@@ -43,3 +44,9 @@ Default.argTypes = {
 export const WithControlledInput = TemplateWithControlledInput.bind({})
 
 export const WithUncontrolledInput = TemplateWithUncontrolledInput.bind({})
+
+export const withIcons = Template.bind({})
+withIcons.args = {
```

Comment:
```suggestion
export const WithIcons = Template.bind({})
WithIcons.args = {
```
Just spotted a small typo. Let's keep these name consistent